soc/ite/ec/it8xxx2/Kconfig.soc

```diff
@@ -36,45 +23,44 @@ config SOC_IT8XXX2_REG_SET_V2
 	  This option is selected by a variable of which soc, and will
 	  determine the register for the IT82xx2 specification.
 
-if SOC_IT8XXX2
-
-choice IT8XXX2_SERIES
-	prompt "IT8XXX2 Series"
-	default SOC_IT81302_BX
-
 config SOC_IT81302_BX
-	bool "IT81302 BX version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V1
 
 config SOC_IT81202_BX
-	bool "IT81202 BX version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V1
 
 config SOC_IT81302_CX
-	bool "IT81302 CX version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V1
 
 config SOC_IT81202_CX
-	bool "IT81202 CX version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V1
 
 config SOC_IT82202_AX
-	bool "IT82202 AX version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V2
 	select SOC_IT8XXX2_EC_BUS_24MHZ if !DT_HAS_ITE_IT82XX2_USB_ENABLED
 
 config SOC_IT82302_AX
-	bool "IT82302 AX version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V2
 	select SOC_IT8XXX2_EC_BUS_24MHZ if !DT_HAS_ITE_IT82XX2_USB_ENABLED
 
 config SOC_IT82002_AW
-	bool "IT82002 AW version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V2
 	select SOC_IT8XXX2_EC_BUS_24MHZ if !DT_HAS_ITE_IT82XX2_USB_ENABLED
 
-endchoice
-
 config SOC_IT8XXX2_PLL_FLASH_48M
 	bool "Flash frequency is 48MHz"
 	default y
```

Comment:
can add `select FLASH` to this, seems it is needed but only for this

Reply:
done